### PR TITLE
Feature/framed connection

### PR DESCRIPTION
**Pull request by faulnegx**
Add framed options for connection and pool. Seems to work well. Let me know if this is good.

### CODE REVIEW

File: aiothrift/pool.py [hunk 1]
```diff
@@ -44,7 +55,7 @@ class ThriftPool:
     """Thrift connection pool.
     """
 
-    def __init__(self, service, address, *, minsize, maxsize, timeout=None):
+    def __init__(self, service, address, *, minsize, maxsize, timeout=None, framed):
```
Comment (ryanwang520) [inline aiothrift/pool.py]: positional argument `framed` is not valid after keyword argument `timeout` .

Comment (faulnegx) [reply aiothrift/pool.py]: Fixed

**Comment by nanassito** [comment]
It appears that this was not released on pypi. Is that on purpose ?

**Comment by ryanwang520** [comment]
Published https://pypi.org/project/aiothrift/

**Comment by nanassito** [comment]
Thank you @ryanwang520

**Pull request merged**